README.md

```diff
@@ -123,6 +123,13 @@ pagrant:
     php_fpm_version: '7.2' # Define the PHP version. Default is '7.1'.
     nodejs_version: '8.x' # Define the NodeJS major version. Default is '6.x'.
     nginx_sites_default_root: /app/public # Absolute path of the public dir. Default is '/app/web'.
+    apt_packages:
+      pkg:
+        - npm
```

Comment:
put the defaults here (and mention it as comment)
This is designed to be copy-pasted so don't break that

Reply:
In fact better make different section for it